object_types/openshift/pod_log.yml

```diff
@@ -0,0 +1,77 @@
+version: 2016.06.21.0
+
+defaults:
+  type: string
+  doc_values: true
+
+pod_log:
+  type: group
+  description: >
+    Contains common fields, this is what populates the actual the mappings of the _default_.
+  fields:
+    - name: kubernetes
+      type: group
+      description: >
+        kubernetes-specific metadata.
+      fields:
+      - name: pod_name
+        type: string
+        index: not_analyzed
+        description: >
+          The name of the pod
+        norms:
+          enabled: True
+
+      - name: labels
+        type: group
+        description: >
+          Labels attached to the OpenShift object
+          Each label name is a subfield of labels field.
+          Each label name is de-dotted: dots in the name are replaced with
+          underscores.
+
+      - name: pod_id
+        type: string
+        index: not_analyzed
+        description: >
+          TODO
+        norms:
+          enabled: True
+
+      - name: namespace_name
+        type: string
+        index: not_analyzed
+        description: >
+          The name of the namespace
+        norms:
+          enabled: True
+
+      - name: namespace_id
+        type: string
+        index: not_analyzed
+        description: >
+          The name of the namespace
+        norms:
+          enabled: True
+
+      - name: hostname
```

Comment:
Does this have to be `hostname`?  It's hardcoded to `host` in the fluentd k8s meta filter.  I understand if it should be `hostname` for consistency with the other `hostname` fields.